src/architect/account/account.utils.ts

```diff
@@ -14,10 +14,22 @@ export default class AccountUtils {
     }),
   };
 
-  static async getAccount(app: AppService, account_name?: string, account_message?: string): Promise<Account> {
+  static getLocalAccount(): Account {
+    // Account ids are UUID so there is no chance of collision
+    return {
+      id: 'dev',
+      name: 'dev (Local Machine)',
```

Comment:
Nice! I don't mind this solution and I can still have a default account for every other cmd.